packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -98,9 +73,17 @@ export async function processOutboundDocumentQueryResps({
       },
     });
 
-    const docRefsWithMetriportId = filterDocRefsWithMetriportId(docRefs);
+    const responseWithDocsToDownload = await getRespWithDocsToDownload({
```

Comment:
Updated the flow so now we loop through each gwResp, we then check if they have docs that need downloading (not in our system) and if they do not we filter out the gw completely

Reply:
nit: plural on "responses"